web/template/partial/stream/chat/chat.gohtml

```diff
@@ -39,6 +46,86 @@
                 </div>
             </div>
         </div>
+        <!-- Active Poll UI -->
+        <template x-if="activePoll && !admin">
```

Comment:
this should also be hidden when the chat is collapsed

Reply:
yes i def. missed that!! Thanks! 😅